e2e/e2e_suite_test.go

```diff
@@ -33,17 +34,24 @@ var tmpPath = NewTmpPath()
 var _ = SynchronizedBeforeSuite(func() []byte {
 	tmpPath.SetPath(path.Dir(shell.CurrentDir())+"/dist", os.Getenv("PATH"))
 	ensureRequiredBinaries()
+
+	if skip, found := os.LookupEnv("IKE_SKIP_CLUSTER_SHUTDOWN"); found {
```

Comment:
Should we align the names of these vars, e.g. all are IKE_E2E prefixed?

Reply:
```suggestion
	if skip, found := os.LookupEnv("IKE_E2E_SKIP_CLUSTER_SHUTDOWN"); found {
```